apps/text-extraction/requirements.txt

```diff
@@ -0,0 +1,4 @@
+beautifulsoup4
+langchain 
```

Comment:
I think we want this https://github.com/langchain-ai/langchain/blob/e106e9602fb0085e9f11cb792382cf647ce07929/libs/text-splitters/pyproject.toml#L10 . langchain will pull a lot more.

On pypi https://pypi.org/project/langchain-text-splitters/

Reply:
Then there are these html splitters too.
https://python.langchain.com/api_reference/text_splitters/html.html